neqo-common/src/codec.rs

```diff
@@ -472,6 +475,17 @@ impl From<Encoder> for Vec<u8> {
     }
 }
 
+impl Write for Encoder {
```

Comment:
`std::fmt::Write` has just one method to implement: `fn write_str(&mut self, s: &str) -> Result<(), Error>`